materialize-snowflake/sqlgen.go

```diff
@@ -266,6 +289,14 @@ WHEN NOT MATCHED THEN
 		r.{{$key.Identifier -}}
 	{{- end -}}
 );
+{{ end }}
+
+{{ define "copyHistory" }}
+SELECT FILE_NAME, STATUS, FIRST_ERROR_MESSAGE FROM TABLE(INFORMATION_SCHEMA.COPY_HISTORY(
+  TABLE_NAME=>'{{ $.Table.Identifier }}',
+  START_TIME=>DATEADD(MINUTE, -5, TO_TIMESTAMP_LTZ('{{ $.StartTime }}'))
```

Comment:
Even though I am fetching the start time from Snowflake itself, and I'm doing that before we even begin sending the pipe requests, I hit this issue a few times where the start time ended up being after the start time of the pipe load requests we sent... so I ended up with this -5 minutes. It's a bit gross though.